src/components/TopPick/TopPick.tsx

```diff
@@ -67,6 +67,17 @@ const linkStyles = css`
     }
 `;
 
+// to override a tag styles from dangerouslySetInnerHTML
+const inCommentLinkStyling = css`
+    a {
+        color: ${palette.brand[500]};
```

Comment:
We moved away from using the `palette` object to try and reduce bundles size. You can get `brand` with 

```
import { brand } from '@guardian/src-foundations/palette';
```

Obvs, in this case, `palette` is already imported but that should, ideally, be refactored away at some point so it's good to stick to the convention to make that refactor easier

Reply:
Thanks, have updated!